File: Chalkable.Web/app/chlk/models/grading/AlternateScore.js
REQUIRE('chlk.models.id.AlternateScoreId');

NAMESPACE('chlk.models.grading', function () {
    "use strict";

    /** @class chlk.models.grading.AlternateScore*/
    CLASS('AlternateScore', [
        chlk.models.id.AlternateScoreId, 'id',
        String, 'name',
        String, 'description',

        [ria.serialize.SerializeProperty('includeinaverage')],
        Boolean, 'includeInAverage',

        [ria.serialize.SerializeProperty('percentofmaximumscore')],
        Number, 'percentOfMaximumScore'
    ]);
});

File: Chalkable.StiConnector/SyncModel/AlternateScore.cs
namespace Chalkable.StiConnector.SyncModel
{
    using System;
    
    public class AlternateScore
    {
        public int AlternateScoreID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IncludeInAverage { get; set; }
        public decimal? PercentOfMaximumScore { get; set; }
        public Guid RowVersion { get; set; }
        public Guid DistrictGuid { get; set; }
    }
}
